site/profile/manifests/app/pipelines/agent.pp

```diff
@@ -1,15 +1,28 @@
 class profile::app::pipelines::agent (
-  String $access_token = 'fake',
-  String $secret_key = 'fake',
+  String $access_token,
+  String $secret_key,
+  String $download_url,
 ){
 
-  if $::kernel == 'windows' {
-    fail('Unsupported OS')
-  }
+#  if ($access_token == 'ENTER_VALUE') or ($access_token == 'fake'){
+#    fail('You must enter access_token, secret_key, and download_url if you want to install the Pipelines agent.')
+#  }
```

Comment:
Can pull this block - don't need to check for required parameters, Puppet does it for us now